internal/kubebuilder/cmdutil/cmdutil.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
If I understand the Apache 2.0 license correctly, we need to keep the original licenses and attributions for all the files that we copied (or substantially copied) from Kubebuilder.

With the exception of necessary changes (e.g. the default file modes), these files should be copied verbatim to make future merges/patches easier.